.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,67 @@
+name: Bug Report
```

Comment:
This could be more like "Report an issue"?

Reply:
ok. doing it